src/xenia/kernel/util/xex2.cc

```diff
@@ -869,6 +876,22 @@ int xe_xex2_load_pe(xe_xex2_ref xex) {
     xex->sections->push_back(section);
   }
 
+  /*
+  if (header->export_table_offset) {
```

Comment:
it is, but i commented it out because it's an absolute address so it's not reliable if we decide to relocate the module.